couler/argo.py

```diff
@@ -0,0 +1,186 @@
+import atexit
```

Comment:
can we add the license here ?

Reply:
See https://github.com/couler-proj/couler/pull/2 for LICENSE document. We can add license headers to each file later in a separate PR so it's clear what's changed based off internal code.